pom.xml

```diff
@@ -631,7 +631,8 @@
           <excludes>
             <exclude>**/*TestCase.java</exclude>
             <exclude>**/*IT.java</exclude>
-            <exclude>**/*Keycloak*.java</exclude>
+            <!--no longer exclude keycloak as travis should be running it
```

Comment:
@ryandawsonuk same.. remove the comments.. they are dead code that we don't want to maintain